c7n/mu.py

```diff
@@ -639,6 +639,10 @@ def name(self):
         return "custodian-%s" % self.policy.name
 
     @property
+    def log(self):
```

Comment:
is this used anywhere? if not probably should be removed. self.log in the codebase typically refers to an actual logger instance.